packages/playwright-test/src/reporters/base.ts

```diff
@@ -96,7 +98,7 @@ export class BaseReporter implements Reporter  {
   }
 
   onError(error: TestError) {
-    console.log('\n' + formatError(this.config, error, colors.enabled).message);
+    this._fatalErrors.push(error);
```

Comment:
This can be perceived as a regression by some!

Reply:
This code has moved to dot, line and list to not interfere with their internal line counting details.